backend/JavaWebService/src/main/java/com/kainos/ea/resources/CapabilitiesController.java

```diff
@@ -0,0 +1,27 @@
+package com.kainos.ea.resources;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import java.util.List;
+
+@Path("/api")
+public class CapabilitiesController {
+    private CapabilitiesService service;
+
+    public CapabilitiesController(CapabilitiesService service){
+        this.service = service;
+    }
+
+    @GET
+    @Path("/capabilities")
+    @Produces(MediaType.APPLICATION_JSON)
+    public List<Capability> getCapabilities(){ return service.getCapabilities(); }
+
+    @GET
+    @Path("/capabilities/{capabilityId}")
+    @Produces(MediaType.APPLICATION_JSON)
+    public List<Role> getRolesForCapability(@PathParam("capabilityId") int capabilityId){ return service.rolesPerCapability(capabilityId); }
+}
```

Comment:
I would encourage you to get into the habit of checking for line-breaks at the end of your files. GitHub shows this symbol as a warning because if you don't use a line break, some presentations and processings of the file can get very awkward. IntelliJ can do this for you automatically:
![image](https://user-images.githubusercontent.com/38751994/131362183-9d230147-93ab-4a04-8935-12b0293e8c02.png)